.github/workflows/msrv.yml

```diff
@@ -18,6 +20,20 @@ jobs:
 
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/cache@v2
+        with:
+          path: |
+            ~/.cargo/bin
+            ~/.cargo/registry/index
+            ~/.cargo/registry/cache
+            ~/.cargo/git/db
+          key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
```

Comment:
nit: we don't have a `Cargo.lock` (because it's a library), so this does not evaluate:
<img width="338" alt="image" src="https://github.com/Eppo-exp/rust-sdk/assets/1366419/97a377ac-0fa9-4ac7-8b0f-baa1cc223761">